app/assets/stylesheets/application/donations/donations.scss

```diff
@@ -63,6 +88,17 @@
   }
 }
 
+#new_donation {
```

Comment:
Avoid using id selectors<br>Selector `new_donation` should be written in lowercase with hyphens